Modules/install-jaspBase.R.in

```diff
@@ -39,17 +42,48 @@ if (md5SumsChanged(modulePkg, moduleLibrary)) {
   options(renv.config.install.verbose = TRUE)
   renv::install(project = modulePkg, library = moduleLibrary, prompt = prompt)
 
-  correctlyInstalled <- installModulePkg(modulePkg, moduleLibrary, prompt, cacheAble=TRUE) 
+  correctlyInstalled <- installModulePkg(modulePkg, moduleLibrary, prompt, cacheAble = TRUE)
 
   if (correctlyInstalled)
     writeMd5Sums(modulePkg, moduleLibrary)
 
 
 }
 
+# compute and cache all hashes and dependencies of all jasp modules
+commitHashObject <- createCommitHashObject(
+  jaspRoot = "@PROJECT_SOURCE_DIR@",
+  file     = file.path("@R_LIBRARY_PATH@", "jaspBase", "cachedCommitInfo.rds")
+)
+
+if (needsInstallation) {
+
+  installedPath <- file.path("@R_LIBRARY_PATH@", "jaspBase")
+  # normalizePath resolves the symlink
+  cachePath <- normalizePath(installedPath)
+
+  # construct the cache path with the custom hash
+  jaspBaseVersion <- commitHashObject[["descriptions"]][["jaspBase"]][["Version"]]
+  jaspBaseHash    <- commitHashObject[["commitHashes"]][["jaspBase"]]
+  newJaspBaseCachePath <- file.path("/home/dvdb/GitHub/jasp/build-jasp-desktop-Desktop_Qt_6_3_1_GCC_64bit-Debug/Modules/renv-cache", "v5", "jaspBase", jaspBaseVersion, jaspBaseHash)
```

Comment:
This is not good...

Needs to be a var or something

Reply:
my bad, should be fixed now!